racket-cas/racket-cas.rkt

```diff
@@ -918,11 +948,12 @@
 
 ; divide u by v
 (define (Oslash: u v)
-  (math-match* (u v)
-    [(r  0) +nan.0]
+  (match* (u v)
+    [((num: r)  0) +nan.0]
     [(u  1) u]
```

Comment:
Why change this?

Reply:
With math-match*, I'll get an error for the pattern `((TimesTerms (== v) us ...) _)`
`match: incorrect use of ... in pattern in: ...`